tests/simple/testdata/broken.textproto

```diff
@@ -91,31 +91,38 @@ section {
   name: "integer_math.textproto/simple_math"
   description: "tests for integer math."
   test {
-    name: "int64_overflow"
+    name: "int64_overflow_positive"
     description: "LLONG_MAX plus one."
     expr: "9223372036854775807 + 1"
     ## Expected behavior: return error for overflow.
     value: { int64_value: -9223372036854775808 }
   }
   test {
-    name: "int64_underflow"
+    name: "int64_overflow_negative"
     description: "LLONG_MIN minus one."
     expr: "-9223372036854775808 - 1"
-    ## Expected behavior: return error for underflow.
+    ## Expected behavior: return error for overflow.
     value: { int64_value: 9223372036854775807 }
   }
   test {
-    name: "uint64_overflow"
+    name: "uint64_overflow_positive"
     description: "ULLONG_MAX plus one."
     expr: "18446744073709551615u + 1u"
     ## Expected behavior: return error for overflow.
     value: { uint64_value: 0 }
   }
   test {
-    name: "uint64_underflow"
+    name: "uint64_overflow_negative"
     description: "zero minus one."
     expr: "0u - 1u"
-    ## Expected behavior: return error for underflow.
+    ## Expected behavior: return error for overflow.
     value: { uint64_value: 18446744073709551615 }
   }
+  test {
+    name: "fp_underflow"
+    description: DBL_MIN(2^-1074) divided by two"
+    expr: "1e-324  / 2.0"
+    ## Expected behavior: return error for underflow.
```

Comment:
No, we don't expect error here - it should be the closest number to the exact result (2^-1075), which in this case is zero.

Reply:
OK, move it back to tests.